src/repository/postDAO.js

```diff
@@ -0,0 +1,27 @@
+const uuid = require("uuid");
+const { PutCommand } = require("@aws-sdk/lib-dynamodb");
+
+const tableName = "";
+
+async function createPost(username, text, score){
```

Comment:
This would be a service file function because it is handling user data. I would separate it from the DAO